test/valgrind/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:latest
+
+ARG work_dir=/ bundle_dir=/usr/local/bundle
+
+WORKDIR ${work_dir}
+# RUN bundle config set --local path ${bundle_dir}
+
+RUN apt update && apt install -yqq libgeos-dev valgrind
+
+COPY Gemfile rgeo.gemspec ./
+COPY lib/rgeo/version.rb ./lib/rgeo/
+RUN bundle install
+
+COPY . .
+RUN rake compile
+
+
+CMD ["rake", "test:valgrind"]
```

Comment:
Is this just for convenience of testing? Or does RubyMemcheck specify that we use docker

Reply:
This is related to `bin/mac_valgrind`, which I could actually rename `bin/valgrind` maybe? This allows users to run the `rake test:valgrind` on a computer that cannot have valgrind

Reply:
ahh yeah maybe we rename it to `bin/valgrind` or `bin/docker_valgrind`. But doesn't matter too much.